index.html

```diff
@@ -5,8 +5,44 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet"/>
 </head>
 <body>
-    
+    <section class= "main_header">
+        <h1>Weather Report</h1>
+        <h3>For the wonderful city:
+            <span id="headerCity" class="header_city"></span
+        ></h3>
```

Comment:
The closing angle bracket here is a little odd on its own line. We sometimes format tags like this if there are many attributes, but on a closing tag, we should keep it with the rest of the tag.